coremltools/models/neural_network.py

```diff
@@ -2495,10 +2495,10 @@ def set_pre_processing_parameters(self, image_input_names = [], is_bgr = False,
         Parameters
         ----------
         image_input_names: [str]
-            Name if input blobs that are images
+            Name of input blobs that are images
 
         is_bgr: boolean | dict()
-            Image pixel order (RGB or BGR)
+            Channel order for input blobs that are images. BGR if True, RGB if False.
```

Comment:
Do we want to explain here about if it's a dict?

Reply:
Updated.